tests/auto-placement.test.html

```diff
@@ -0,0 +1,28 @@
+<html>
+  <body>
+    <div data-ea-publisher="test" data-ea-manual="false"></div>
+
+    <script type="module">
+      import { expect } from "@open-wc/testing";
+      import { runTests } from "@web/test-runner-mocha";
+      import { default as sinon } from "sinon";
+
+      import { wait, Placement } from "../index";
+
+      let stub = sinon.stub(Placement.prototype, "fetch");
+      const response_html = "<div></div>";
+      const elem_placement = document.createElement("div");
+      elem_placement.innerHTML = response_html;
+      stub.resolves(elem_placement.firstChild);
```

Comment:
@davidfischer this is what mocking the fetch method looks like, using `sinon` for mocking. It should avoid any calls to the EA server, but there could be some timing related behavior too, assuming the request happens before the mock/stub is in place.